src/olympia/addons/utils.py

```diff
@@ -247,3 +259,130 @@ def get_error_message(self):
         except IndexError:
             msg = None
         return msg
+
+
+class PermissionEnablerCreator:
+    """Helper class to create new permission enabler add-ons and versions.
+
+    Assumes parameters have already been validated beforehand."""
+
+    SITE_PERMISSIONS = {
+        'webmidi': 'WebMIDI',
```

Comment:
New name is actually `midi-sysex` (there are 2 permissions behind the scenes, but the only one we'll care about to start is that new one)